### ISSUE TITLE
android crashes when you pass empty string as track url

### ISSUE THREAD

**Issue by punksta** [react-native-kit/react-native-track-player#368]
```
        await TrackPlayer.reset();

	await TrackPlayer.add({
			id: "sample",
			url: "",
			title: "sample",
			artist: "sample"
		});
        await TrackPlayer.play();
```

```
  "react-native-track-player": {
      "version": "git+https://github.com/react-native-kit/react-native-track-player.git#871e5cfcf8b350282a0fd964b576f9f1cf4c5d6c",
      "from": "git+https://github.com/react-native-kit/react-native-track-player.git#dev"
    },
```

crash is related to native url parsing

**Comment by Guichaguri** [comment]
That's expected. We can add a more user-friendly error, but we will never allow passing empty URLs as tracks which doesn't make sense at all.

**Comment by punksta** [comment]
I think same. But it shouln't just crash on java level.

**Comment by Guichaguri** [comment]
Fixed in ebc76f172c21a04871bc4ab8d1c4c5093bef03d9
